### DISCUSSION TITLE
Download appears to be working

### DISCUSSION THREAD

**Discussion by xezoxeon** [aclap-dev/video-downloadhelper#1399] [Help and Q&A]
### Your question

Website:soaper.tv
Download Status: Active
Download rate: Under 100kB/s (would like to be like 1mb at least lol)

It would appear something has changed; as the downloads appear to be "working again"

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"a1c80bf5b445417c"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\Admin\\Documents"
        }
    },
    "concurrent_downloads_max": 11,
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_wide_ui": true,
    "open_count_store": 5
}

**Comment by xezoxeon** [comment]
Update: Speeds of 6.24 MB/s
Multi download high speed with a vpn of 1mbs+/download 

High speed for the high sails :D

**Comment by Mattmjs** [comment]
You have an updated version of the extension that's what changed, looks like chrome & edge had it available since last week.
Firefox is lagging and hasn't got it yet.

**Reply by xezoxeon** [reply]
Nice! I have checked each day and this is first its worked! no issues currently.
